@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/targeting/fn_targetingSched.sqf

```diff
@@ -42,4 +42,7 @@ if (inputAction "vehLockTargets" > 0.5 && fza_ah64_locktargstate == 0 && count _
 };
 if (inputAction "vehLockTargets" < 0.5 && fza_ah64_locktargstate == 1) then {
     fza_ah64_locktargstate = 0;
+};
+if (inputAction "lockTarget" > 0.5) then {
+    fza_ah64_mycurrenttarget = cursorTarget;
```

Comment:
Why is this required? I am trying to keep `fza_ah64_mycurrenttarget` to be just targets selected by the FCR.

Reply:
nessesery for deselection targets as there is not other method and troubleshooing/ bugfixing with shooting gun through fcr/fcr